web/src/main/webapp/WEB-INF/jsp/team/match/detail.jsp

```diff
@@ -0,0 +1,120 @@
+                                         </tbody>
+                                     </table>
+                                 </div>
+                             </div>
+                             <div class="col-md-6">
+                                 <div class="table-div table-responsive">
+                                     <table class="table table-fixed table-hover table-striped table-fixed-small">
+                                         <thead>
+                                         <tr class="tr-fixed-small">
+                                             <th>Time</th>
+                                             <th>Scorer</th>
+                                             <th>Assist</th>
+                                         </tr>
+                                         </thead>
+                                         <tbody>
+                                         <c:forEach items="awayGoals" var="awayGoal">
+                                             <tr class="tr-fixed-small clickable-row" data-href="/match/${awayGoal.id}">
+                                                 <td>${awayGoal.matchTime.format(formatter)}</td>
+                                                 <td>${awayGoal.scorer.surname}</td>
+                                                 <td>${awayGoal.assistant.surname}</td>
+                                             </tr>
+                                         </c:forEach>
+                                         </tbody>
+                                     </table>
+                                     <a href="<c:url value="/match/newgoal">" class="btn btn-default btn-link pull-right" data-dismiss="modal">Add Goal</a>
+                                 </div>
+                                 <div class="p-y-1"></div>
+                                 <p class="m-a-0">Roster</p>
+                                 <div class="table-responsive">
+                                     <table class="table table-hover table-striped">
+                                         <thead>
+                                         <c:forEach items="awayPlayers" var="awayPlayer">
+                                         <tr>
+                                             <td>${awayPlayer.firstname}</td>
+                                             <td>${awayPlayer.surname}</td>
+                                         </tr>
+                                         </c:forEach>
+                                         </tbody>
+                                     </table>
+                                 </div>
+                             </div>
+                         </div>
+                     </div>
+                     <div class="m-t-1">
+                         <a href="match_form.html" class="btn btn-success">Edit</a>
+                         <a href="#myModal" class="btn btn-success m-l-1" data-toggle="modal" role="button">Delete</a>
```

Comment:
pokud ten modal ma fungovat, musíš to zinicializovat, aby ten javascript je začal zpracovávat